### PR TITLE
Add JDK 19

### PR DESCRIPTION
**Pull request by Goooler**
https://www.azul.com/downloads/?version=java-19-sts&os=macos&package=jdk

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,4 +30,5 @@ brew install <name>
 | OpenJDK 16 | `zulu-jdk16` | 16.0.2 | [![JDK16](https://github.com/mdogan/homebrew-zulu/workflows/JDK16/badge.svg)](https://github.com/mdogan/homebrew-zulu/actions) |
 | OpenJDK 17 | `zulu-jdk17` | 17.0.4.1 | [![JDK17](https://github.com/mdogan/homebrew-zulu/workflows/JDK17/badge.svg)](https://github.com/mdogan/homebrew-zulu/actions) |
 | OpenJDK 18 | `zulu-jdk18` | 18.0.2.1 | [![JDK18](https://github.com/mdogan/homebrew-zulu/workflows/JDK18/badge.svg)](https://github.com/mdogan/homebrew-zulu/actions) |
+| OpenJDK 18 | `zulu-jdk19` | 19.0.0 | [![JDK18](https://github.com/mdogan/homebrew-zulu/workflows/JDK19/badge.svg)](https://github.com/mdogan/homebrew-zulu/actions) |
```
Comment (Goooler) [inline README.md]:
```suggestion
| OpenJDK 18 | `zulu-jdk19` | 19.0.0 | [![JDK19](https://github.com/mdogan/homebrew-zulu/workflows/JDK19/badge.svg)](https://github.com/mdogan/homebrew-zulu/actions) |
```

**Comment by Goooler** [comment]
Not sure why badge dot work

image

**Comment by mdogan** [comment]
Hmm.. because workflow is named in lowercase but image url has uppercase `JDK19`.  No worries, [Upgrade Casks](https://github.com/mdogan/homebrew-zulu/actions/workflows/upgrade.yml) task will fix it when it runs soon. Normally you don't even need to send a PR for new/updated JDKs, it automatically creates a PR for them.

**Comment by Goooler** [comment]
Gotcha.

**Pull request merged**